README.md

```diff
@@ -97,17 +97,22 @@ broken RPM build environment. You should add this to your ~/.rpmmacros:
 
 ## Creating WINDOWS INSTALLER
 
-Install Cygwin and include at least the ghostscript, zip, dos2unix and libgcc
+Install Cygwin and include at least the gcc, make, cygrunsrv, ghostscript, zip, dos2unix and libgcc
```

Comment:
Thanks for this. More precisely, it should be `gcc-core` rather than `gcc`, as exemplified later.